pom.xml

```diff
@@ -586,8 +591,9 @@
         <nimbus.jose.jwt.version>7.9</nimbus.jose.jwt.version>
         <org.json.version>3.0.0.wso2v1</org.json.version>
         <os.mvn.plugin.version>1.6.2</os.mvn.plugin.version>
-        <otel.alpha.version>1.6.0-alpha</otel.alpha.version>
-        <otel.version>1.6.0</otel.version>
+        <otel.alpha.version>1.16.0-alpha</otel.alpha.version>
```

Comment:
what if we bring in the latest releases? Did we test?

Reply:
@VirajSalaka By the time I created this PR (9th Aug), the latest version was `1.16`. Now `1.17` and `1.18` are also available. I'll update to `1.18` and test the feature.

Reply:
Updated otel related libs to `1.18`